ansible/roles/operator-pipeline/templates/openshift/tasks/show-support-link.yml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: show-support-link
+spec:
+  params:
+    - name: env
+      description: Enviroment. One of [dev, qa, stage, production]
+    - name: type
+      description: Connect case type. 
+      default: "CERT"
+    - name: source
+      description: Connect case source.
+      default: "tekton"
+    - name: cert_project_id
+      description: Pyxis certification project id.
+    - name: cert_project_type
+      description: Connect cert project type. Should be Operator Bundle Image for operator pipelines.
+      default: "Operator Bundle Image"
+  steps:
+    - name: show-support-link
+      image: quay.io/redhat-isv/operator-pipelines-images
+      script: |
+        #! /usr/bin/env bash
```

Comment:
Wouldn't it be good to add  `set -e` here also?

Reply:
Added